From c375b638b55ff16f15423445f47a6b9eb10d90c4 Mon Sep 17 00:00:00 2001
From: Limeng <Meng.Li@windriver.com>
Date: Tue, 8 Jan 2019 16:46:15 +0800
Subject: [PATCH 2/2] driver: pcie: add uncorrectable error macro define

When run kdump feature, uncorrectable error is triggered during
kernel crashes down. So, it is need add uncorrectable error into
macro MSGF_MISC_SR_MASKALL so that this error is cleared in
driver probe.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/pci/controller/pcie-xilinx-nwl.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/pci/controller/pcie-xilinx-nwl.c b/drivers/pci/controller/pcie-xilinx-nwl.c
index 867ff69..09733b2 100644
--- a/drivers/pci/controller/pcie-xilinx-nwl.c
+++ b/drivers/pci/controller/pcie-xilinx-nwl.c
@@ -79,6 +79,7 @@
 /* Misc interrupt status mask bits */
 #define MSGF_MISC_SR_RXMSG_AVAIL	BIT(0)
 #define MSGF_MISC_SR_RXMSG_OVER		BIT(1)
+#define MSGF_MISC_SR_UNCORRECT_ERR	BIT(3)
 #define MSGF_MISC_SR_SLAVE_ERR		BIT(4)
 #define MSGF_MISC_SR_MASTER_ERR		BIT(5)
 #define MSGF_MISC_SR_I_ADDR_ERR		BIT(6)
@@ -95,6 +96,7 @@
 
 #define MSGF_MISC_SR_MASKALL		(MSGF_MISC_SR_RXMSG_AVAIL | \
 					MSGF_MISC_SR_RXMSG_OVER | \
+					MSGF_MISC_SR_UNCORRECT_ERR | \
 					MSGF_MISC_SR_SLAVE_ERR | \
 					MSGF_MISC_SR_MASTER_ERR | \
 					MSGF_MISC_SR_I_ADDR_ERR | \
-- 
1.7.5.4

